### ISSUE TITLE
expand height on hover to show full title

### ISSUE THREAD

**Issue by z1haze** [fullcalendar/fullcalendar#6725]
### Checklist

Please mark these items with an [x]

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

It would be great if there was an option to trigger the height to scale to fit the full description on hover

**Comment by acerix** [comment]
Do you mean if the content doesn't fit in an event it should expand to show it all? It would likely work better to show the full details in a popup instead of resizing the event:

https://fullcalendar.io/docs/event-tooltip-demo

**Comment by z1haze** [comment]
Yes, a popup is certainly an option, I just think for a ux perspective it might be cleaner and easier if when you mouseover the event, it could expand down to show the full details of that event, without having an obtrusive modal or something each time you want to read event details. I imagine that the height animation would not push elements below it down, but rather float on top of ones below it (covering them) as they are all absolutely positioned.

**Comment by robel93** [comment]
I would like this function aswell!
